tests/integration/test_health_checks.py

```
@@ -0,0 +1,160 @@
+        # Verify unhealthy status detected
+        assert health_status["status"] == "unhealthy"
+        assert "checks" in health_status
+        assert health_status["checks"]["docker_contexts"]["status"] == "fail"
+
+    async def test_health_check_timeout(self, docker_mcp_config):
+        """Test health check timeout handling."""
+        mock_context_manager = Mock(spec=DockerContextManager)
+        mock_context_manager.config = docker_mcp_config
+
+        # Simulate slow operation that times out
+        async def slow_operation():
```

Ruff (0.14.4)

Missing return type annotation for private function `slow_operation`

Add return type annotation: `str`

(ANN202)